pkg/kube/sourcerepositories.go

```diff
@@ -39,13 +40,42 @@ func GetRepositoryGitURL(s *v1.SourceRepository) (string, error) {
 	return spec.HTTPCloneURL, nil
 }
 
-// FindSourceRepository returns a SourceRepository for the given namespace, owner and name
-// or returns an error if it cannot be found
-func FindSourceRepository(jxClient versioned.Interface, ns string, owner string, name string) (*v1.SourceRepository, error) {
+// FindSourceRepositoryWithoutProvider returns a SourceRepository for the given namespace, owner and repo name.
```

Comment:
👍 I am wondering though whether `FindSourceRepositoryByName` (menaing CRD name) would be a more fitting name. The pair of ``FindSourceRepositoryByName` and `FindSourceRepositoryByLabels` would then make more sense together.

Reply:
Fine by me - I have an unconscious tendency to try retain existing names even when that’s silly.